Resources/Prototypes/Entities/Objects/Power/powercells.yml

```diff
@@ -210,10 +275,19 @@
         - type: PowerCellVisualizer
           prefix: m_hi
 
+- type: entity
+  id: PowerCellMediumHighPrinted
+  parent: PowerCellMediumHigh
```

Comment:
give this a suffix or make it nospawn. same with all the other printed variants

Reply:
Done